sample-apps/react/egress-composite/src/components/LogoAndTitleOverlay.tsx

```diff
@@ -12,7 +12,7 @@ export const LogoAndTitleOverlay = () => {
       data-testid="logo-and-title-overlay"
       className="eca__logo-and-title-overlay"
     >
-      {titleText?.length && (
+      {!!titleText?.length && (
```

Comment:
```suggestion
      {titleText && (
```

Why do we need to check the `.lenght` at all?

Reply:
Good point, adjusted.